src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -248,8 +248,6 @@ type ErrorCode =
     | UserDefinedTypeInEntryPointSignature = 6233
     | InnerTupleInEntryPointArgument = 6234
     | ArrayOfArrayInEntryPointArgument = 6235
-    | OtherEntryPointExists = 6236
-    | MultipleEntryPoints = 6237
```

Comment:
It's unlikely that anyone is using these directly, but technically they should be marked `Obsolete` first before being removed.

Reply:
done